webplugin/js/app/mck-app.js

```diff
@@ -142,6 +145,35 @@ function ApplozicSidebox() {
             return false;
         }
     };
+    function addScriptInsideHtml() {
+        var script =  
+        "function showAfterLoad(){"+
```

Comment:
@ab14bhardwaj this looks confusing and difficult to maintain, any other solution for this? see if writing it as a separate function and then using function.toString() works?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/toString